docker-compose.yml

```diff
@@ -84,3 +84,11 @@ services:
       - 2456:2456/udp
     depends_on:
       - boiler
+  migrate:
+    image: alpine 
+    container_name: migrate
+    volumes:
+      - ./migrate:/data
+    entrypoint: ["/bin/sh", "-c", "apk add --no-cache curl && chmod +x /data/* && /data/valheim.sh"]
+    depends_on:
```

Comment:
Right now this spins up and shuts down, but leaves stoker and postgres running in the background. How do we actually want this to behave?

Reply:
That behavior is fine, though I'd like for `valheim.sh` to have the correct permissions in git so that we don't have to `chmod +x` it each time and I'd like to use `wget` so we don't have to install `curl` each time